From 05357d4bf9631a381b58c35cc1adb36198d35b49 Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Sat, 17 Nov 2012 05:08:56 +0200
Subject: [PATCH 07/11] Revert "Hotkeys for sort modes and for New Empty File"

This reverts commit daf7118742889c4997aca99880ca70b2d4955506.

Unfortunately, Ctrl+F1 thru Ctrl+F4 are often used by some of
window managers to switch workspaces and some of them even don't
allow to change that binding so it goes into conflict.
---
 src/main-win-ui.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/main-win-ui.c b/src/main-win-ui.c
index a12b38e..ae8c0a5 100644
--- a/src/main-win-ui.c
+++ b/src/main-win-ui.c
@@ -177,7 +177,7 @@ static GtkActionEntry main_win_actions[]=
     {"Reload2", NULL, NULL, "<Ctrl>R", NULL, G_CALLBACK(on_reload)},
     /* for popup menu - these entries exist in FmFolderView popup still */
     {"NewFolder", "folder", N_("Folder"), "<Ctrl><Shift>N", NULL, G_CALLBACK(bounce_action)},
-    {"NewBlank", "text-x-generic", N_("Blank File"), "<Ctrl><Alt>N", NULL, G_CALLBACK(bounce_action)}
+    {"NewBlank", "text-x-generic", N_("Blank File"), NULL, NULL, G_CALLBACK(bounce_action)}
 };
 
 /* main_win_toggle_actions+main_win_mode_actions - see 'ViewMenu' for mnemonics */
@@ -206,10 +206,10 @@ static GtkRadioActionEntry main_win_sort_type_actions[]=
 
 static GtkRadioActionEntry main_win_sort_by_actions[]=
 {
-    {"ByName", NULL, N_("By _Name"), "<Ctrl>F1", NULL, COL_FILE_NAME},
-    {"ByMTime", NULL, N_("By _Modification Time"), "<Ctrl>F2", NULL, COL_FILE_MTIME},
-    {"BySize", NULL, N_("By _Size"), "<Ctrl>F3", NULL, COL_FILE_SIZE},
-    {"ByType", NULL, N_("By File _Type"), "<Ctrl>F4", NULL, COL_FILE_DESC}
+    {"ByName", NULL, N_("By _Name"), NULL, NULL, COL_FILE_NAME},
+    {"ByMTime", NULL, N_("By _Modification Time"), NULL, NULL, COL_FILE_MTIME},
+    {"BySize", NULL, N_("By _Size"), NULL, NULL, COL_FILE_SIZE},
+    {"ByType", NULL, N_("By File _Type"), NULL, NULL, COL_FILE_DESC}
 };
 
 static GtkRadioActionEntry main_win_side_bar_mode_actions[]=
-- 
1.8.0.1

